packages/integration-tests/suites/tracing/browsertracing/backgroundtab/subject.js

```diff
@@ -0,0 +1,11 @@
+document.getElementById('go-background').addEventListener('click', () => {
+  Object.defineProperty(document, 'hidden', { value: true, writable: true });
+  const ev = document.createEvent('Event');
+  ev.initEvent('visibilitychange');
+  document.dispatchEvent(ev);
```

Comment:
ah this isn't the nicest, but I guess there isn't really a better way around it.